File: RedSheet.DbEntities/Models/vLiteOurTeamMemberRecord.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteOurTeamMemberRecords", Schema = "dbo")]
	[Serializable]
	public partial class vLiteOurTeamMemberRecord  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int LiteOurTeamMemberId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string LiteOurTeamMemberName { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int LiteProjectBackgroundId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int PersonalityId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string Position { get; set; } 

	public vLiteOurTeamMemberRecord()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-lite-our-team-member-record.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteOurTeamMemberRecord {
    constructor(vLiteOurTeamMemberRecord?: vLiteOurTeamMemberRecord )  {
        let properties = [ "liteOurTeamMemberId", "liteOurTeamMemberName", "liteProjectBackgroundId", "personalityId", "position",];
        for (let property of properties)
            if (vLiteOurTeamMemberRecord && vLiteOurTeamMemberRecord[property])
                this[property] = vLiteOurTeamMemberRecord[property];
    }
 
	liteOurTeamMemberId : number =   0 ;
 
    @required()
    @maxLength(50)
	liteOurTeamMemberName : string =   undefined;
 
    @range(1,2147483647)
	liteProjectBackgroundId : number =   undefined;
 
    @range(1,2147483647)
	personalityId : number =   undefined;
 
    @required()
    @maxLength(50)
	position : string =   undefined;


}
